### PR TITLE
Add a beta label

### LINKED ISSUE #248: Labelling pages with "beta"
We should have the ability to mark pages released in our stable environment with `beta` or equivalent in order for us to communicate to our users this is a new feature which should not be expected to be completed ( and also maybe that we especially wants feedback on it )

### PR DESCRIPTION
**Pull request by frode-aarstad**
**Issue**
Resolves #248 

## Pre review checklist

- [x] Added appropriate labels

### CODE REVIEW

File: webviz_ert/plugins/_observation_analyzer.py [hunk 1]
```diff
@@ -12,14 +12,21 @@
...
+    def __init__(self, app: dash.Dash, project_identifier: str, beta: bool = False):
         super().__init__(app, project_identifier)
         self.set_callbacks(app)
+        self.beta = beta
 
     @property
     def layout(self) -> Component:
         return html.Div(
             [
+                html.Div(
```
Comment (oysteoh) [inline webviz_ert/plugins/_observation_analyzer.py:24]: I wonder if we should do something more to this text - feels a bit strange only heaving BETA as header... 🤔  Would it make sense having a descriptive text instead / in addition? 
```
* This page is considered a beta version and could be changed or removed. You are encouraged to use it and give feedback to us regarding functionality and / or bugs. 
```

**Comment by oysteoh** [comment]
Looks good ! One minor comment regarding the text and how we are visualizing this is a beta to make it more clear for the users.

**Review by oysteoh — APPROVED** [review]
I think this looks good now ! Informative message to the user 👍 👍  I'll approve but we should get the jenkins test to also pass - that is currently not possible but will be as soon as bleeding is rebuilt.

**Comment by frode-aarstad** [comment]
Jenkins test this please

**Pull request merged**